src/main/webapp/AddSchoolData.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>School Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <p>Add A Course Below</p>
+    <form action="/add-info" method="POST">
+        <p>School Name</p>
+        <textarea name="school-name">SchoolName</textarea>
+        <br><br>
+        <p>Course Name</p>
+        <textarea name="course-name">CourseName</textarea>
+        <br><br>
```

Comment:
Just a small note: I found it more intuitive to leave these text areas blank and give them labels, so we don't have to delete the "SchoolName", "CourseName", existing info every time before adding a course. Not something you need to do, just figured I'd mention that in case you found it easier to work with!